src/neo/Trie/IKVStore.cs

```diff
@@ -0,0 +1,7 @@
+namespace Neo.Trie
+{
+    public interface IKVStore : IKVReadOnlyStore
```

Comment:
Where is the implementation of this interface?

Reply:
None. One want to use the Trie should implement the interface. For example, when I integrate trie in blockchain storage.

Reply:
https://github.com/neo-project/neo/blob/master/src/neo/Persistence/IStore.cs

I think we should use this interface.

Reply:
I think usually we modify store through `ISnapshot`, `IStore` can be used to read directly.

Reply:
Agree with share the same interface